reflex/components/overlay/menu.py

```diff
@@ -96,10 +97,13 @@ def create(
             children = []
 
             if button:
-                children.append(MenuButton.create(button))
+                if not isinstance(button, (MenuButton, Button)):
+                    children.append(MenuButton.create(button))
+                else:
+                    children.append(button)
             if not items:
                 items = []
-            children.append(MenuList.create(*items))
+            children.append(MenuList.create(items=items))
```

Comment:
what is the advantage of adding another arg, `items`, here?

Reply:
It's to avoid conflict with the "manual" API where users defined every component by themselves.

(Like in `reflex-web` where the menu items in the navbar are defined manually.)